### ISSUE TITLE
Can i use your project to make a little bit of Money

### ISSUE THREAD

**Issue by CoGaJoe21** [beforeblazergithub/BlazerGames#3]
Hi I am Joe
I want to use your project as a sample for a Game Website.
I also wanted to make a little bit of Money with that. I can give you Credits and link your channel.
I would change the title to SISIGames. Sisi is my Cat. And i want to put Google Ads on the website.
Thanks for an answer. And your project is awsome.
Have a nice day
Joe

**Comment by beforeblazergithub** [comment]
Yes you can use my project to make money.
Thanks for asking me!
- Blazer

**Comment by CoGaJoe21** [comment]
Hi Blazer
Are the Games Open Source that youve linked in your video.
Like i want to make money with google adsense so are they copyrighted
Thanks for an anwser
Joe

**Comment by beforeblazergithub** [comment]
Nope, they aren't copyrighted at all, it's for everyone to use freely as
they wish.
-Blazer

**Comment by CoGaJoe21** [comment]
Hi Blazer its me. Joe
I have a problem with the games like it only shows 10 games and then the games just becomes text.
Could you help me.
Joe

**Comment by CoGaJoe21** [comment]
Sorry i found why it didnt worked thanks for the code by the way. (I dont really speak english)

**Comment by spiderman1212** [comment]
Blazer put me on with sum huzz

**Comment by spiderman1212** [comment]
u know u got no chance
cs u spend ur free time coding

**Comment by spiderman1212** [comment]
Im js plyin blazer srry if i offended u
